zerver/tests/test_realm.py

```diff
@@ -901,7 +901,8 @@ def test_creating_realm_creates_system_groups(self) -> None:
             "@role:everyone",
             "@role:internet",
         ]
-        self.assertEqual(user_group_names.sort(), expected_system_group_names.sort())
+        # works the same as asserEqual but it doesn't need the list to be sorted
```

Comment:
`assertCountEqual` is a standard library function and we already have several uses of it; I don’t think it needs to be re-documented in a code comment.

Reply:
You are right, I will keep that in mind next time! I updated the file. Thank you!